crates/ollama/src/ollama.rs

```diff
@@ -357,6 +367,108 @@ pub async fn show_model(
     Ok(details)
 }
 
+#[cfg(any(test, feature = "test-support"))]
+pub mod fake {
```

Comment:
I'd prefer use of the more general FakeHttpClient.  I am not sure why I don't see more use of it.